reference/progress.v1.yaml

```diff
@@ -0,0 +1,670 @@
+        - id
+      properties:
+        id:
+          type: string
+        display:
+          type: object
+          description: 'The human readable representation of the Verb in one or more languages. This does not have any impact on the meaning of the Statement, but serves to give a human-readable display of the meaning already determined by the chosen Verb.'
+      example:
+        id: 'http://example.com/xapi/verbs#defenestrated'
+        display:
+          en-US: defenestrated
+          es: defenestrado
+    Activity:
+      title: XAPI Activity
+      type: object
+      properties:
+        name:
+          type: object
+        description:
+          type: object
+        type:
+          type: object
+        moreInfo:
+          type: object
+        extensions:
+          type: object
+    StatementReference:
+      title: XAPI Statement Reference
+      type: object
+      required:
+        - objectType
+        - id
+      properties:
+        objectType:
+          type: string
+        id:
+          type: string
+    Result:
+      title: XAPI Result
+      type: object
+      properties:
+        score:
+          $ref: '#/components/schemas/Score'
+        success:
+          type: boolean
+        completion:
+          type: boolean
+        response:
+          type: string
+        duration:
+          type: string
+        extensions:
+          type: object
+    Score:
+      title: XAPI Score
+      type: object
+      properties:
+        scaled:
+          type: number
+        raw:
+          type: number
+        min:
+          type: number
+        max:
+          type: number
+    Context:
+      title: XAPI Context
+      type: object
+      properties:
+        registration:
+          type: string
+        instructor:
+          $ref: '#/components/schemas/Agent'
+        team:
+          type: object
+        contextActivities:
+          type: object
+        revision:
+          type: string
+        platform:
+          type: string
+        language:
+          type: string
+        statement:
+          $ref: '#/components/schemas/Statement'
+        extensions:
+          type: object
+    Attachment:
+      title: XAPI Attachment
+      type: object
+      required:
+        - usageType
+        - display
+        - contentType
+        - length
+        - sha2
+      properties:
+        usageType:
+          type: string
+        display:
+          type: object
+        description:
+          type: object
+        contentType:
+          type: string
+        length:
+          type: integer
+        sha2:
+          type: string
+        fileUrl:
+          type: string
+    SimpleProgress:
+      title: SimpleProgress
+      type: object
+      properties:
+        id:
+          type: string
+        activityId:
+          type: string
+        studentEckId:
+          type: string
+          description: The eckid of the student for which the progress is reported.
+        studentId:
```

Comment:
Wondering which value would be expected for studentId - I guess the Learning Application internal studentId doesnt add any value for other parties. Isnt the studentEckId sufficient?

Reply:
probably : yes. The student id would be the same as the id recieved from the SIS API. Until ECKId is unique available this is still a double identifier

Reply:
Why did we not made this attribute smart like studentId . Now you have to look at the context of the request or result.

Can we please change within the API's the id that means this is a student to StudentId !